CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc

```diff
@@ -60,7 +60,7 @@ void CaloTPGTranscoderULUT::loadHCALCompress(HcalLutMetadata const& lutMetadata,
 	      (not HcalGenericDetId(*i).isHcalZDCDetId()) and
 	      (not HcalGenericDetId(*i).isHcalCastorDetId()))
 	    edm::LogWarning("CaloTPGTranscoderULUT") << "Encountered invalid HcalDetId " << HcalGenericDetId(*i);
-	    continue;
```

Comment:
@gartung I think not, as I said above. Maybe @bsunanda (who seems to be the last contributor?) can comment?